core/src/com/projectkorra/projectkorra/earthbending/metal/Extraction.java

```diff
@@ -39,6 +46,12 @@ public Extraction(final Player player) {
 		this.tripleChance = getConfig().getDouble("Abilities.Earth.Extraction.TripleLootChance");
 		this.cooldown = getConfig().getLong("Abilities.Earth.Extraction.Cooldown");
 		this.selectRange = getConfig().getInt("Abilities.Earth.Extraction.SelectRange");
+		this.ironGolemDrops = getConfig().getInt("Abilities.Earth.Extraction.IronGolem.Drops");
+		this.ironGolemDamage = getConfig().getDouble("Abilities.Earth.Extraction.IronGolem.Damage");
+		this.ironGolemDropMaterial = Material.getMaterial(getConfig().getString("Abilities.Earth.Extraction.IronGolem.DropMaterial"));
+		if (this.ironGolemDropMaterial == null) {
+			this.ironGolemDropMaterial = Material.IRON_INGOT;
+		}
```

Comment:
is there a reason this doesn't default to the config default option? (Iron nugget)

Reply:
No reason. You are right, though. 

However, I feel that making this configurable makes no sense. Perhaps we could make the drop item from IRON GOLEMS as something like a Bernoulli, where the probability/chance of dropping ingots `p` (IRON INGOT) is defined in the config (like 40; 40%) and the probability of dropping nuggets would be `100 - p` (60% here). It gives users the freedom to also choose exactly what material it drops by setting the chance to either 0 (always nuggets) or 100 (always ingots).

Reply:
- Addressed in: https://github.com/ProjectKorra/ProjectKorra/pull/1333/commits/579f52d0a7535946e16b23fcf29abcfbccfb6916
   - `Extraction` on IRON GOLEMS will now only drop iron nuggets.